src/pynwb/ecephys.py

```diff
@@ -19,20 +19,25 @@ class ElectrodeGroup(NWBContainer):
     __nwbfields__ = ('name',
                      'description',
                      'location',
-                     'device')
+                     'device',
+                     {'name': 'spike_event_series', 'doc': 'doc', 'child': False})
```

Comment:
This should be an _ElectricalSeries_, and _doc_ should be something that makes sense. The docstring will get rendered on the API docs, so this is what users will see if they use the API docs for reference.

Reply:
Yeah I can change the doc, I just wanted to draw your attention to the error so we can fix it. Why should this be an ElectricalSeries?